src/desktop_notifier/dbus.py

```diff
@@ -64,13 +47,10 @@ class DBusDesktopNotifier(DesktopNotifierBase):
 
     supported_hint_signatures = {"a{sv}", "a{ss}"}
 
-    def __init__(
-        self,
-        app_name: str = "Python",
-        notification_limit: int | None = None,
-    ) -> None:
-        super().__init__(app_name, notification_limit)
+    def __init__(self, app_name: str) -> None:
+        super().__init__(app_name)
         self.interface: ProxyInterface | None = None
+        self._platform_nid_to_identifier: bidict[int, str] = bidict()
```

Comment:
seem to have lost a comment in my review so just restating that to a code reader `nid` is very unclear (even to me, who has spent some time with the code)... i assume it means `notification_id`? in which case i would just call it `notification_id` which makes it abundantly clear exactly what it is.

Reply:
Fair point. Done.